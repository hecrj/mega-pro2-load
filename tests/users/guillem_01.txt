14

10
20
30
40
50
60
70
80
90
100
110
120
100
140

7

5 3 6 0 0 1 0 0 2 0 7 0 4 0 0

10
2 1 13
50
2 8 13
50
2 5 1
40
2 7 13
50
3 9 13 8
60
2 11 13
70
2 2 13

-1 13 100
-6
