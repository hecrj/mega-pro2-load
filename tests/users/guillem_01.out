Peticion procesada y servidores
0 1
2 5 

